cmd/imagebuilder/imagebuilder.go

```diff
@@ -90,10 +90,14 @@ func main() {
 	options.Out, options.ErrOut = os.Stdout, os.Stderr
 	authConfigurations, err := docker.NewAuthConfigurationsFromDockerCfg()
 	if err != nil {
-		log.Fatalf("reading authentication configurations: %v", err)
+		if strings.Contains(err.Error(), "no such file or directory") {
```

Comment:
Can you check this versus the constant rather then the string.

Something like.

if errors.Is(err, syscall.ENOENT)

Reply:
Thanks for the review. I have fixed this, but I do not know why my changes broke the Travis CI:
```
0.16s$ docker pull busybox
Using default tag: latest
Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
The command "docker pull busybox" failed and exited with 1 during .
```

Reply:
You didn't break it.  I'm not sure why it worked before, but there was no point where the Travis configuration was starting the daemon after stopping it and attempting to update it.  I've opened https://github.com/openshift/imagebuilder/pull/291 to fix that.  You can cherry-pick it onto your PR if you like, or rebase after it's merged to pick it up.

Reply:
Thanks @nalind, I will rebase after it's merged.